src/tabs/presets/presets.js

```diff
@@ -292,7 +293,9 @@ TABS.presets.tryLoadPresets = function() {
     this._divGlobalLoading.toggle(true);
     this._domWarningNotOfficialSource.toggle(!this.presetsSourcesDialog.isOfficialActive);
 
-    this.presetsRepo.loadIndex().then(() => {
+    this.presetsRepo.loadIndex()
+    .then(() => this.checkPresetSourceVersion())
```

Comment:
This is a sync operation, why is it wrapped  inpromise and `then`? What is value in that?

Reply:
not really sync because of the `GUI.showYesNoDialog(dialogSettings)` inside

Reply:
Gotcha